tophat/yvm
# Refactor `which` command and add color to messages.

## Description


- Add `chalk` dependency
- Add `error` and `success` coloured log functions
- Extract logic for getting yvm version from PATH into helper function
- Update exit code for error states in `which` command


## DevQA

### DevQA Prep

- Please `make node_modules`, this PR uses a new package

### DevQA Steps


- Run `yvm which` in the various use cases
	- Verify that errors are shown in red
	- Verify that message for matching version in .yvmrc is shown in green

### Comments:


- This is a WIP
